src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/read_ir/read_ir.cpp

```diff
@@ -11,7 +11,7 @@
 
 #include "shared_test_classes/base/utils/ranges.hpp"
 #include "shared_test_classes/base/utils/generate_inputs.hpp"
-#include "ngraph_functions/builders.hpp"
+#include "ov_models/builders.hpp"
```

Comment:
@itikhono Could you please chack, do we really need to use this header in the file? I believe it is just not removed artifact :(